client/src/lobby/nav.ts

```diff
@@ -94,10 +146,11 @@ export const init = () => {
 
 const initTooltips = () => {
   const tooltips = [
-    'create-game',
-    'replay',
-    'resources',
-    'settings',
```

Comment:
why are you renaming these

Reply:
I wanted to `initToolTips` also hit `nav-buttons-pregame-change-variant`

Reply:
using a `games-` prefix implies that they are part of the game screen, when they are not
why not use a `lobby-` prefix

Reply:
The `games-` prefix isn't by me, was there before, but I can rename them all to `lobby-`. Agree it makes more sense.

Reply:
18ce2e41dcfebb2467ee20a4cf317c861452502a does this renaming. this is a fairly deep change... hope it doesn't break anything (though it seems to work as far as my testing can see)